### ISSUE TITLE
Partials vs. Layouts

### ISSUE THREAD

**Issue by lyzadanger** [cloudfour/drizzle-builder#56]
Right now, the builder has a number of `src` objects that defines where and how it builds parsed data. The default `src` options are:

```
  src: {
    data    : {
      basedir: 'src/data',
      glob: 'src/data/**/*'
    },
    layouts : {
      basedir: 'src/layouts',
      glob: 'src/layouts/**/*'
    },
    pages   : {
      basedir: 'src/pages',
      glob: 'src/pages/**/*'
    },
    partials: {
      basedir: 'src/partials',
      glob: 'src/partials/**/*'
    },
    patterns: {
      basedir: 'src/patterns',
      glob: 'src/patterns/**/*.html'
    }
  }
```

**I wanted to examine `partials` and `layouts` to determine whether they are _both_ truly needed.** The rough current semantic explanation is that `partials` are, well, just partials, while `layouts` are partials but also more layout-like and intended for use with `handlebars-layouts`. It's also a holdover from `fabricator`, and I didn't really think deeply about it.

Currently: In the `prepare` phase, anything matched by the `partials` glob is registered as a partial on the `handlebars` instance. Additionally, anything matching the `layouts` glob is _also_ registered as a partial. Thus, they are treated the same here.

In the `parse` phase, however, `layouts` are additionally added to a `layouts` property on the global `drizzleData` object—this becomes necessary for accessing them later in the `render` phase for `handlebars-layouts` reasons. I can expand on _why_ that is necessary if y'all are curious but will leave it out here for brevity.

All that aside, I'm wondering if having separate `partials` really does anything for us. In a "combined" model, all of the things would get registered as partials and all of the things would be available on the global data object. I'm not sure what to call these things if we combine them, maybe `templates`?

Anyway, can you think of any reason that we should keep `partials` separate from `layouts`?

/cc @tylersticka @mrgerardorodriguez @saralohr

**Comment by tylersticka** [comment]
> Anyway, can you think of any reason that we should keep partials separate from layouts?

No, not really. I'd be fine with this. :+1:

**Comment by erikjung** [comment]
:+1:

**Comment by mrgerardorodriguez** [comment]
:+1:

**Comment by lyzadanger** [comment]
Fixed in #58
